content/blog/inheritance/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Inheritance
+date: "2019-04-17T21:33:32.169Z"
+description: Animals are awesome.
```

Comment:
Cool! This shows up on the main page, so can we add a good description for all of the posts you added perhaps?

*Looks at own blog*

Yeah I'm a hypocrite.